pos_pricelist/static/src/js/widgets.js

```diff
@@ -66,7 +72,7 @@ function pos_pricelist_widgets(instance, module) {
         }
     });
 
-    module.PosBaseWidget.include({
+    screens.PosBaseWidget.include({
```

Comment:
PosBaseWidget can be reached as following:

     var PosBaseWidget = require('point_of_sale.BaseWidget')